test/fail_compilation/b19523.d

```diff
@@ -0,0 +1,19 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/b19523.d(12): Error: undefined identifier `SomeStruct`
+fail_compilation/b19523.d(13): Error: function `b19523.foo(int delegate() arg)` is not callable using argument types `(_error_)`
```

Comment:
Should `_error_` really appear in the user-facing output?

Reply:
I would say ideally not.